web/spacewalk-web.changes.eth.multi-sst

```diff
@@ -0,0 +1 @@
+- Fix sticky header scroll bug
```

Comment:
Hi @Etheryte, sorry for the nitpicking here, but mentioning a bug in a .changes without the bsc sounds misleading. as user/customer I'll try to look up for that bug.
What about something like this?
```suggestion
- Fix sticky header infinite scroll
```